platform/app/pluginConfig.json

```diff
@@ -96,7 +96,7 @@
     },
     {
       "packageName": "dicom-microscopy-viewer",
-      "importPath": "/dicom-microscopy-viewer/dicomMicroscopyViewer.min.js",
+      "importPath": "/dicomMicroscopyViewer.min.js",
```

Comment:
can you explain why you need this change?

Reply:
This is not how the dynamic-import works with DMV. Instesad of ```/dist/dynamic-import/dicom-microscopy-viewer/dicomMicroscopyViewer.min.js``` its just ```/dist/dynamic-import/dicomMicroscopyViewer.min.js```. At some point in time Bill made this change to dmv https://github.com/ImagingDataCommons/dicom-microscopy-viewer/pull/113 that modified the webpack to create this folder under dynamic-import but the commit was reverted later on after some problems with linking.

Reply:
@wayfarer3130 is this correct?

Reply:
Ideally just allowing it to be "./dicomMicroscopyViewer.min.js would be good so that it works with a PUBLIC_URL
The problem with NOT having the sub-directory is that the build creates several files which overlap name wise with other applications, specifically the codecs library.  Fixing the linking issues to work with a sub-path would really be optimal as it would prevent naming collisions.

Reply:
Ok, but I don't think this is a problem with DMV, but how is OHIF using it, no? To me, it's weird to have ```dmv/dist/dynamic-import/dmv(again)/build.js``` and I'm not sure how the names overlap? If some files produced by DMV overlap, then I think I should update DMV so the names are definitely unique (e.g,. append version to the file name).

Reply:
Looks like all the changes were reverted, not just these ones.  This is not going to load successfully for the cornerstone version, and there isn't a way to have two versions.  Some mixture of the changes is going to be needed, rather than a total revert of that PR.

Reply:
Ok, I'll schedule a call to discuss these changes in detail. I'm not sure why these conflicts are happening. Maybe the problem might be elsewhere.